### DISCUSSION TITLE
Initial Data Population

### DISCUSSION THREAD

**Discussion by ramkishore-yerubandi** [vendurehq/vendure#1232] [Q&A]
1. Looking for an option where I want to populate only counties, zones only.
2. Is there a way to set the default currency code to 'INR' for the default channel while creating the project?

**Comment by michaelbromley** [comment]
Hi,

To populate only countries & zones, you can use the [InitialData](https://www.vendure.io/docs/developer-guide/importing-product-data/#initial-data) object and leave the other properties empty.

There's currently no built-in way to set the currency code for the default channel. But you can create a script to do so.
